patience-deck-fi.ts

```diff
@@ -0,0 +1,331 @@
+        <location filename="../qml/pages/components/GameOverOverlay.qml" line="68"/>
+        <source>You win</source>
+        <extracomment>Shown on top of the game when player has won the game</extracomment>
+        <translation>Voitit</translation>
+    </message>
+    <message id="patience-la-game_over">
+        <location filename="../qml/pages/components/GameOverOverlay.qml" line="71"/>
+        <source>Game over</source>
+        <extracomment>Shown on top of the game when player has lost the game</extracomment>
+        <translation>Peli päättyi</translation>
+    </message>
+    <message id="patience-bt-new_game">
+        <location filename="../qml/pages/components/GameOverOverlay.qml" line="95"/>
+        <location filename="../qml/pages/OptionsAndRules.qml" line="79"/>
+        <source>New game</source>
+        <extracomment>Starts a new game</extracomment>
+        <translation>Uusi peli</translation>
+    </message>
+    <message id="patience-bt-restart">
+        <location filename="../qml/pages/components/GameOverOverlay.qml" line="103"/>
+        <location filename="../qml/toolbar/Toolbar.qml" line="446"/>
+        <source>Restart</source>
+        <translation>Aloita alusta</translation>
+    </message>
+    <message id="patience-me-about">
+        <location filename="../qml/pages/Game.qml" line="101"/>
+        <source>About</source>
+        <translation>Tietoja</translation>
+    </message>
+    <message id="patience-me-select_game">
+        <location filename="../qml/pages/Game.qml" line="107"/>
+        <source>Select game</source>
+        <translation>Valitse peli</translation>
+    </message>
+    <message id="patience-me-game_options">
+        <location filename="../qml/pages/Game.qml" line="114"/>
+        <source>Options &amp; Rules</source>
+        <translation>Valinnat ja säännöt</translation>
+    </message>
+    <message id="patience-me-common-settings">
+        <location filename="../qml/pages/OptionsAndRules.qml" line="42"/>
+        <source>Common settings</source>
+        <translation>Yhteiset asetukset</translation>
+    </message>
+    <message id="patience-he-options">
+        <location filename="../qml/pages/OptionsAndRules.qml" line="56"/>
+        <source>Options &amp; Rules</source>
+        <translation>Valinnat ja säännöt</translation>
+    </message>
+    <message id="patience-la-starting_new_or_changing_resuffle">
+        <location filename="../qml/pages/OptionsAndRules.qml" line="62"/>
+        <source>Starting a new game or changing game options will resuffle the cards</source>
+        <translation>Uuden pelin aloittaminen ja asetusten muuttaminen sekoittaa kortit uudestaan</translation>
+    </message>
+    <message id="patience-la-starting_new_resuffle">
+        <location filename="../qml/pages/OptionsAndRules.qml" line="64"/>
+        <source>Starting a new game will resuffle the cards</source>
+        <translation>Pelin aloittaminen uudestaan sekoittaa kortit</translation>
```

Comment:
Probably should be "Uuden pelin aloittaminen" instead of "Pelin aloittaminen uudestaan" as we are not restarting here.